socket_server/utils/helpers.js

```diff
@@ -0,0 +1,122 @@
+
+
+module.exports = {
+  findWinner: (obj1, obj2) => {
```

Comment:
A great refactor is to leverage OOP imo. 

Creating a class of RPS and having all the logic held in there.

Then just instantiating a new instance and passing in relevant data